models/backbones/abstract_backbone.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+import torch.nn as nn
+
+
+class AbstractBackbone(nn.Module, ABC):
+
+    @abstractmethod
+    def forward(self, x):
```

Comment:
Due to the the implementation of the  `nn.Module` class there is no need to define abstract class.

```Python
class Module:
    ...
    forward: Callable[..., Any] = _forward_unimplemented
    ...
    def _forward_unimplemented(self, *input: Any) -> None:
            raise NotImplementedError('smth...')
```

Reply:
removed ABC from baseclasses